File: C++/task3/main.cpp
#include <iostream>
#include <vector>
#include "cube.hpp"
#include "stacking.cpp"

int main() {
    cube cube1=cube("red", 5);
    cube cube2=cube("red", 6);
    cube cube3=cube("blue", 5);

    std::vector<cube*> cubeList = {&cube1, &cube2, &cube3};
    std::cout << "The maximum height is " << calculate_height(stack_cubes(cubeList)) << std::endl;
}

File: Python/task3/main.py
#!/usr/bin/python3

from cube import Cube
from stacking import stack_cubes

def main():
    """main: Driver function to make sure code is running correctly """
    cube1 = Cube('red', 5)
    cube2 = Cube('red', 6)
    cube3 = Cube('blue', 5)
    cube_list = [cube1, cube2, cube3]
    print(stack_cubes(cube_list))

main()
